packages/backend/src/controllers/users/users.controller.ts

```diff
@@ -51,7 +51,7 @@ export class UsersController {
     const houseDoc = await this.houseStoreService.findOne(userDoc.house);
```

Comment:
I think the typing is problematic on these methods, they are capable of returning nullish values yet the typing indicates they are guaranteed to exist.